### PR TITLE
Adding PagerDuty config to Prod.py

### PR DESCRIPTION
**Pull request by AshleyPoole**
This adds the PagerDuty config to the prod.py and env.prod.example files -- there was a timing issue with PRs being merged which meant the original prod config PR went in after PagerDuty support PR. This is leading to the app to fail to start up when in PROD mode as mention by @Dgadavin on PR #25 .

### CODE REVIEW

**Comment by AshleyPoole** [comment]
Ready for review @evnsio @adamlphillips

File: env.prod.example [hunk 1]
```diff
@@ -9,4 +9,9 @@ DB_PORT="<SET_THIS>"
...
 DB_PASSWORD="<SET_THIS>"
 SITE_URL="<SET_THIS>"
-DJANGO_SETTINGS_MODULE="response.settings.prod"
\ No newline at end of file
+DJANGO_SETTINGS_MODULE="response.settings.prod"
+ENCRYPTED_FIELD_KEY="<SET_THIS>"
+PAGERDUTY_ENABLED="<True|False>"
+PAGERDUTY_API_KEY="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
+PAGERDUTY_SERVICE="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
+PAGERDUTY_DEFAULT_EMAIL="<SET_THIS_IF_PAGERDUTY_IS_ENABLED>"
```
Comment (adamlphillips) [inline env.prod.example]: Can you add a newline to make github happy?

File: response/settings/prod.py [hunk 2]
```diff
@@ -55,3 +55,11 @@
...
 except:
     INCIDENT_CHANNEL_ID = None
+
+
+PAGERDUTY_ENABLED = os.getenv("PAGERDUTY_ENABLED") in ("True", "\"True\"", "true", "\"true\"", True, 1)
+if PAGERDUTY_ENABLED:
+    print("PagerDuty is Enabled")
+    PAGERDUTY_API_KEY = get_env_var("PAGERDUTY_API_KEY")
+    PAGERDUTY_SERVICE = get_env_var("PAGERDUTY_SERVICE")
+    PAGERDUTY_DEFAULT_EMAIL = get_env_var("PAGERDUTY_DEFAULT_EMAIL")
```
Comment (adamlphillips) [inline response/settings/prod.py]: Can you add a newline to make github happy?

File: env.prod.example (hunk 1 above)
Comment (AshleyPoole) [reply env.prod.example]: @adamlphillips Added.

File: response/settings/prod.py (hunk 2 above)
Comment (AshleyPoole) [reply response/settings/prod.py]: @adamlphillips Added.

**Pull request merged**